WebIDL/current-realm.html

```diff
@@ -43,6 +43,16 @@
      }, val[0])
    })
 
+   ;["Request", "Response"].forEach(val => {
```

Comment:
You can just loop over `[Request, Response]`?

Reply:
How would that work obtaining them from a different global?

Reply:
I misread `self[0]` as `self`, nevermind.